### PR TITLE
Fix website layout

### PR DESCRIPTION
**Pull request by terencode**

### CODE REVIEW

**Comment by Fohlen** [comment]
So far it looks quiet neat. 
For some reasons the `Post.vue` is broken here, which I will try to fix.
Adding more feedback later

File: src/main.js [hunk 1]
```diff
@@ -36,17 +36,11 @@ const Post = resolve => {
...
-function redirect_to_youtube() {
-  location.href = 'https://www.youtube.com/channel/UCKOcY8wxvWq8pGLcESSpfhw'
-}
-
 const routes = [
     { path: '/home', component: Home , alias: '/' },
     { path: '/people', component: People },
     { path: '/blog', component: Blog },
     { path: '/post/:year/:title', component: Post },
-    { path: '/yt', beforeEnter: redirect_to_youtube },
```
Comment (Fohlen) [inline src/main.js]: Why did you remove the `/yt` shortlink?

File: src/Blog.vue [hunk 2]
```diff
@@ -166,4 +176,9 @@ h4 {
 h4:first-letter {
     text-transform:capitalize;
 }
+.card {
+  margin: 50px;
+  background-color:rgba(50, 50, 50, 0.5);
+  border-radius: 0;
```
Comment (Fohlen) [inline src/Blog.vue]: Wouldn't `border-radius: 5px;` look more nicely?

Comment (terencode) [inline src/Blog.vue]: For sure Fohlen but I saw you applied no border radius to the download button.

File: src/main.js (hunk 1 above)
Comment (terencode) [inline src/main.js]: <s>I HATE HACKS </s> I thought we did not really need this.

**Comment by terencode** [comment]
@Fohlen no I did not see it in the navbar I will fix it now.

File: src/main.js (hunk 1 above)
Comment (Fohlen) [inline src/main.js]: It's on the old website as well and doesn't really do any harm here. Please undo the changes!

**Comment by Fohlen** [comment]
Looks good.
